package.json

```diff
@@ -30,14 +30,22 @@
     "prepublishOnly": "npm test"
   },
   "dependencies": {
+    "@date-io/date-fns": "^1.1.0",
+    "date-fns": "^2.0.0-alpha.27",
+    "final-form": ">=4.12.0",
+    "final-form-arrays": ">=1.1.2",
     "history": "^4.7.2",
     "js-logger": "^1.6.0",
+    "material-ui-pickers": "^2.2.4",
     "prop-types": "^15.5.7",
+    "react-final-form": ">=4.1.0",
+    "react-final-form-arrays": "2.0.1",
```

Comment:
I’d use the `^` version range for all 4 of the final-form dependencies.

Reply:
"react-final-form-arrays" need to be pinned because of this issue: https://github.com/final-form/react-final-form-arrays/issues/72#issuecomment-470603987

Reply:
Ouch! But I’d at least replace `>=4.1.0` with `^4.1.0` for react-final-form. Don’t want users to get 5.0.0 by mistake.

Reply:
Will do!